### PR TITLE
Us10 home

### PR DESCRIPTION
**Pull request by elodiedaubie**

### CODE REVIEW

File: public/assets/css/home.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
+@import url('https://fonts.googleapis.com/css2?family=Londrina+Solid&family=Roboto&family=Skranji&display=swap');
```
Comment (soufcod3) [inline public/assets/css/home.css:1]: This @import is useless (already exists in style.css).
Also as you know, this is a temporary solution as our fonts will be locally hosted in a upcoming commit (from me).

File: src/View/Home/index.html.twig [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
-    <h1>Hello Wilder !</h1>
+        <nav class="nav-home">
+            {#to link a page, add url like example /file/page#}
+            <a href="#" class="questionmark-button"><img src="/assets/images/mobile/question_mark.png" alt="question mark" class="questionmark-icon"></a>
+            <a href="#" class="music-button"><img src="/assets/images/mobile/music_icon.png" alt="music icon" class="music-icon"></a>    
+        </nav>
+    <main>
+        <div class="castle">
+            <picture>
+                <source srcset="/assets/images/desktop/home_castle.png" media="(min-width: 530px)">
```
Comment (soufcod3) [inline src/View/Home/index.html.twig:20]: Looks to me that the source tag can be deleted. 
In my opinion the game will always choose to show the image inside the img tag.
Furthermore the media attribute inside the source tag needs a "href" attribute in order to work which is in our case non existant (cf https://www.w3schools.com/tags/att_a_media.asp).

**Review by soufcod3 — CHANGES_REQUESTED** [review]
2 edits needed before Romain's review :)

File: public/assets/css/home.css (hunk 1 above)
Comment (elodiedaubie) [reply public/assets/css/home.css:1]: Thanks for you feed back, correction done.

File: src/View/Home/index.html.twig (hunk 2 above)
Comment (elodiedaubie) [reply src/View/Home/index.html.twig:20]: Hi Soufiane, actually the tag Picture need both source (which is choose by the web browser only if the media query is respected) and img src (which is choose in all other cases). This syntax correspond to the official documentation https://developer.mozilla.org/fr/docs/Web/HTML/Element/picture

Comment (soufcod3) [reply src/View/Home/index.html.twig:20]: Thanks for the link. It's a nice idea after all! All good

**Review by soufcod3 — APPROVED** [review]

**Review by RomainClair — APPROVED** [review]

**Pull request merged**